test/blocks/z-pattern/z-pattern.test.js

```diff
@@ -0,0 +1,39 @@
+/* eslint-disable no-unused-expressions */
+/* global describe it */
+
+import { readFile } from '@web/test-runner-commands';
+import { expect } from '@esm-bundle/chai';
+
+document.body.innerHTML = await readFile({ path: './mocks/body.html' });
+const { default: init } = await import('../../../libs/blocks/z-pattern/z-pattern.js');
+
+describe('z-patterns', () => {
+  const zPatterns = document.querySelectorAll('.z-pattern');
+  zPatterns.forEach((block) => {
+    init(block);
+    describe('z-pattern', () => {
```

Comment:
don't have to change but there's no need to have the `describe` blocks surrounding each `it`